account-decoder/src/lib.rs

```diff
@@ -28,6 +28,7 @@ use {
 
 pub type StringAmount = String;
 pub type StringDecimals = String;
+pub const MAX_BS58_BYTES: usize = 128;
```

Comment:
Help a brother out:
```suggestion
pub const MAX_BASE58_BYTES: usize = 128;
```
BS means something else to most people

Reply:
I dunno, still seems applicable